src/cocotb_tools/runner.py

```diff
@@ -49,12 +49,13 @@
 
 
 def _as_tcl_value(value: str) -> str:
-    # add '\' before special characters and spaces
-    value = _magic_re.sub(r"\\\1", value)
-    value = value.replace("\n", r"\n")
-    value = _space_re.sub(r"\\\1", value)
-    if value[0] == '"':
```

Comment:
A simpler patch is
```suggestion
    if value[:1] == '"':
```
which avoids needing any other changes.